tox.ini

```
@@ -14,29 +14,27 @@ addopts = --cov splash --cov-report term-missing --cov-report xml
 norecursedirs = .* docs requirements
 
 [testenv]
-deps =
-	django22: Django>=2.2,<2.3
-	django30: Django>=3.0,<3.1
-	django31: Django>=3.1,<3.2
-	django32: Django>=3.2,<3.3
-	-r{toxinidir}/requirements/test.txt
-commands =
-	python -Wd -m pytest {posargs}
+deps = 
+    django32: Django>=3.2,<4.0
+    django40: Django>=4.0,<4.1
+    -r{toxinidir}/requirements/test.txt
+commands = 
+    python -Wd -m pytest {posargs}
 
 [testenv:quality]
-setenv=
+setenv = 
     DJANGO_SETTINGS_MODULE=test_settings
 whitelist_externals = 
-	make
-	rm
-	touch
+    make
+    rm
+    touch
 deps = 
-	-r{toxinidir}/requirements/quality.txt
-	-r{toxinidir}/requirements/test.txt
+    -r{toxinidir}/requirements/quality.txt
+    -r{toxinidir}/requirements/test.txt
 commands = 
-	touch tests/__init__.py
-	pylint splash tests test_utils
-	rm tests/__init__.py
-	isort --check-only --recursive tests test_utils splash manage.py setup.py test_settings.py
-	make selfcheck
+    touch tests/__init__.py
+    pylint splash tests test_utils
+    rm tests/__init__.py
+    isort --check-only --recursive tests test_utils splash manage.py setup.py test_settings.py
```

`recursive` flag isn't needed anymore.
```suggestion
    isort --check-only tests test_utils splash manage.py setup.py test_settings.py
```